pkgs/development/python-modules/spotify-cli-linux/default.nix

```
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonApplication, fetchPypi, lyricwikia, dbus-python }:
+buildPythonApplication rec {
+  pname = "spotify-cli-linux";
+  version = "1.6.0";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0slyc3jfrj3rwq8rv6p5aqkw487aw7a87kmf1fb6n4vnvcf08v7w";
+  };
+  propagatedBuildInputs = [ lyricwikia dbus-python ];
+  doCheck = false;
+  meta = with stdenv.lib; {
+    homepage = "https://pwittchen.github.io/spotify-cli-linux/";
+    maintainers = [ maintainers.kmein ];
+    description = "A command line interface to Spotify on Linux.";
+    license = licenses.gpl3;
+    platforms = platforms.linux;
+  };
+}
```

```suggestion
{ stdenv, buildPythonApplication, fetchPypi, isPy3k, lyricwikia, dbus-python }:
buildPythonApplication rec {
  pname = "spotify-cli-linux";
  version = "1.6.0";

  src = fetchPypi {
    inherit pname version;
    sha256 = "0slyc3jfrj3rwq8rv6p5aqkw487aw7a87kmf1fb6n4vnvcf08v7w";
  };

  disabled = !isPy3k;
  propagatedBuildInputs = [ lyricwikia dbus-python ];
  doCheck = false;

  meta = with stdenv.lib; {
    homepage = "https://pwittchen.github.io/spotify-cli-linux/";
    maintainers = [ maintainers.kmein ];
    description = "A command line interface to Spotify on Linux.";
    license = licenses.gpl3;
    platforms = platforms.linux;
  };
}
```

Also only works on Python3.